Lib/cmd.py

```diff
@@ -108,7 +108,15 @@ def cmdloop(self, intro=None):
                 import readline
                 self.old_completer = readline.get_completer()
                 readline.set_completer(self.complete)
-                readline.parse_and_bind(self.completekey+": complete")
+                if 'libedit' in getattr(readline, '__doc__', ''):
+                    if self.completekey == 'tab':
+                        # libedit expects "^I" for tab completion instead of "tab"
+                        readline.parse_and_bind("bind ^I rl_complete")
+                    else:
+                        readline.parse_and_bind("bind "+self.completekey+" rl_complete")
+                else:
+                    readline.parse_and_bind(self.completekey+": complete")
+
```

Comment:
Please make the lines a little shorter (PEP 8 calls for 80 characters).
Maybe something like this (untested):
```suggestion
                if 'libedit' in getattr(readline, '__doc__', ''):
                    if self.completekey == 'tab':
                        # libedit uses "^I" instead of "tab"
                        command_string = "bind ^I rl_complete"
                    else:
                        command_string = f"bind {self.completekey} rl_complete"
                else:
                    command_string = f"{self.completekey}: complete"
                readline.parse_and_bind(command_string)
```

Reply:
```
>>> completekey = r'"\t"'
>>> f"bind {completekey} rl_complete" == "bind "+completekey+" rl_complete"
True
```
I'm sorry for this. I never had this kind of deep-level repo commit before... I don't know if I did it correctly...